.gitignore

```diff
@@ -0,0 +1,8 @@
+v.out
+vidarrtest-regression.json
+v.out
```

Comment:
Why are all these files repeated so many times?

Reply:
I did not look inside this file, actually. It was generated by the migration scripts, I suppose